packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/widgets.jsx

```diff
@@ -67,20 +68,44 @@ export const MultiplierAmountWidget = observer(() => {
     return <AmountWidget {...amount_widget_props} />;
 });
 
-const RadioGroupOptionsWidget = ({ displayed_trade_param, modal_title }) => {
+const RadioGroupOptionsWidget = ({
+    displayed_trade_param,
+    extra_tooltip_message = '',
+    is_disabled = false,
+    should_show_extra_tooltip = false,
+    modal_title,
+}) => {
     const [is_open, setIsOpen] = React.useState(false);
 
     const toggleModal = () => {
+        if (is_disabled) return;
         setIsOpen(!is_open);
     };
 
     return (
         <React.Fragment>
             <RadioGroupOptionsModal is_open={is_open} toggleModal={toggleModal} modal_title={modal_title} />
             <div className='mobile-widget mobile-widget__multiplier-options' onClick={toggleModal}>
-                <div className='mobile-widget__item'>
+                <div
+                    // className={`mobile-widget__item ${is_disabled ? 'mobile-widget__item-disabled' : ''}`}
```

Comment:
@kate-deriv  do we need this comment? :)

Reply:
it relict after applying suggestions, will remove